android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/retailer/Retailer.kt

```diff
@@ -221,31 +237,35 @@ class Retailer {
     @OptIn(ExperimentalCoroutinesApi::class)
     fun orders(
         context: Context,
-        account: Account,
+        tikiAccount: Account,
```

Comment:
why rename account to tikiAccount?

Reply:
because we have two accounts, one from microblink and one class named account, to differentiate those i changed our accounts variables to tikiAccount and the microblink ones to email/retailerAccount